db/blip_handler.go

```diff
@@ -198,6 +198,11 @@ func (bh *blipHandler) handleSubChanges(rq *blip.Message) error {
 		return fmt.Errorf("blipHandler already has an outstanding continous subChanges.  Cannot open another one")
 	}
 
+	// Create ctx if it does not exist or has been closed
+	if bh.changesCtx == nil || bh.changesCtx.Err() != nil {
```

Comment:
I think it might actually be safer for us _not_ to handle `nil` contexts - it's not recommended to pass around nil contexts, and we want to find the code paths that are doing so.

As far as I can tell, this should never be nil, which would save a fair amount of awkward nil checking for `changesCtx`/`changesCtxCancel`.